packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -92,6 +92,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: STRIMZI_FEATURE_GATES
               value: ""
+            - name: STRIMZI_NETWORK_POLICY_GENERATION
+              value: true
```

Comment:
I don't think we want to have it here. You need to if it in the Helm Chart in a way whcih doesn't render it by default.

Reply:
I lokked into the code base on how to not include it in install files. I have made some desired changes